### ISSUE TITLE
Evaluate renaming the stores table

### ISSUE THREAD

**Issue by ozgune** [citusdata/pgconfus-tutorial-multi-tenant#3]
In the multi-tenant example, we use the ``stores`` table to represent tenants. The name store doesn't immediately communicate the concept of a tenant.

Could we rename this table to one of tenants, customers, organizations, or accounts to better align with names that we use in our docs and blog posts?

**Comment by lfittl** [comment]
@ozgune Yep, good point - one thing we could consider is just adding an `organizations` table and sharding by `organization_id`. 

I would recommend we keep `users` as a local coordinator table, since we want emails to be unique.
